packages/core/src/components/table/work-trial.json

```diff
@@ -0,0 +1,618 @@
+[
```

Comment:
The name of the file could have some example or story-example prefix so we would know it has nothing to do with the implementation.

Reply:
True, I'll prefix it:

- [ ] Prefix file to emphasise it's only example data